### ISSUE TITLE
Redis key has unexpected name

### ISSUE THREAD

**Issue by flavio** [vertexclique/docker-volume-redis#2]
Hi, while testing the code I found that the Redis' keys created via this plugin have an unexpected name.
## How to reproduce the problem
- Start with an empty Redis database
- Start the docker-volume-redis daemon
- Start an interactive bash session inside of a docker container using the Redis volume plugin
- Within the container type: `echo "value1" > /redis/key1`
- Check the contents of the Redis database

Expected outcome:
- A key named: `key1`

Actual result:
- key '/var/lib/docker-volumes/_redis/redis/key1'

What do you think is causing the issue? You don't have to provide a bug fix yet, we can just talk about it.

**Comment by vertexclique** [comment]
I am giving absolute path which contains mountpoint as key to redis
If i was used just name instead of mountpoint when inserting, i will get relative path as you have written.
So removing mountpoint path from walk code can solve the problem.

**Comment by flavio** [comment]
Would you mind to fix this issue and write a unit test for it?

**Comment by flavio** [comment]
Also, do you think that writing a unit test would have prevented this issue to "hit production"?

**Comment by vertexclique** [comment]
Yes I think unit tests are revealing those kinds of bugs and problems and so we can detect them before hit to production and probably this issue will be found early if somebody wrote a unit tests for this case before prod.

**Comment by flavio** [comment]
I saw you introduced a test inside of commit 5c02f6777eb875397bcd6221b941134708621746.

However I would have expected a unit test, not an integration one. The unit tests has the advantage of being faster to run and not requiring docker and redis running.

The integration test you wrote is nice, but there's room for improvements; see all the comments I made. Also the unit test should have a `tearDown()` method cleaning up the environment (to stop the redis database at least).

One last thing: it's always better to make separate commits, one per issue addressed and reference the github issue. That makes easier to track the evolution of code, perform rollbacks and also have a nicer discussion on github.
